### DISCUSSION TITLE
Java Agent so11y is a build-in feature, it reports meters automatically after boot.

### DISCUSSION THREAD

**Discussion by 2han9b1n** [apache/skywalking#12836] [Q&A] [answered]
Why does my current version 10.1.0 not display any data on the instrument after startup

**Comment by wu-sheng** [comment, marked as answer]
Because we haven't released 9.4 Java agent, https://github.com/apache/skywalking-java/blob/main/CHANGES.md.

The reason of not-releasing is because of new feature <**Add async-profiler feature for performance analysis**>. This is a new and important on-demand profiling feature. We need OAP 10.2 release, which should be 2 months(after Chinese new year) away at least.
